docs/tutorials/writing_a_new_data_struct.md

```diff
@@ -101,4 +105,202 @@ If everything is hooked together properly, JSON files for your new data struct s
 
 ## Example
 
-TODO
+The following example shows all the pieces that make up the data pipeline for `DecimalSymbolsV1`.
+
+### Data Struct
+
+*components/decimal/src/provider.rs*
```

Comment:
Nit: Should we turn this to an link to the code? Similar for other paths in this "Example" section.